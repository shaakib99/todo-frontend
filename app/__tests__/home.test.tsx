import { cleanup, render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Home from "../page";

afterEach(() => {
  cleanup();
});

describe("<Home />", () => {
  it("Match Snapshot", () => {
    render(<Home />);
    const homeSnapshot = renderer.create(<Home />).toJSON();
    expect(homeSnapshot).toMatchSnapshot();
  });
});
