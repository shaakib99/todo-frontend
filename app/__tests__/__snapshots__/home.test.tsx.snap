// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Match Snapshot 1`] = `
<div
  className="h-screen md:h-full flex gap-1 items-center p-5 justify-center md:justify-between"
>
  <section
    className="h-full w-full flex flex-col items-center justify-center"
  >
    <p
      className="text-4xl font-semibold"
      data-testid="left-title-text"
    >
      Get Your Daily
       
      <span
        className="text-5xl text-primary font-extrabold"
      >
        WORK
      </span>
       
      Done!!
    </p>
    <button
      className="ant-btn css-dev-only-do-not-override-15rg2km ant-btn-default m-5 h-12 px-8 py-2 bg-primary text-md font-semibold"
      disabled={false}
      onClick={[Function]}
      style={
        {
          "border": "none",
          "color": "inherit",
        }
      }
      type="button"
    >
      <span>
        Get Started
      </span>
    </button>
  </section>
  <section
    className="h-full mt-3 w-full hidden md:flex items-center justify-center "
  >
    <div
      className="-rotate-6"
    >
      <div
        width="100%"
      />
    </div>
  </section>
</div>
`;
